app/webpack/src/table-select-all.js

```diff
@@ -1,54 +1,68 @@
+        link.textContent = link.getAttribute('data-copy-clear');
       }
       else {
-        link.addClass('select-all').text(link.data('copy-select'));
+        link.classList.add('select-all');
+        link.textContent = link.getAttribute('data-copy-select');
       }
     }
 
     function setCategory(row) {
-      let vacantItem = row.find(".table-category-vacant");
-      if (row.find('input:checked').length) {
-        vacantItem.addClass("table-category-preview");
+      let vacantItem = row.querySelector(".table-category-vacant");
+      if (row.querySelector('input:checked')) {
+        vacantItem.classList.add("table-category-preview");
       } else {
-        vacantItem.removeClass("table-category-preview");
+        vacantItem.classList.remove("table-category-preview");
       }
     }
 
-    $('.select-clear-all').click(function() {
-      const table = $(this).parents('table');
-      if($(this).hasClass('clear-all')) {
-        $("#screen-reader-messages").text("All transactions deselected.");  //this adds a message to the message div which is read out by the screen reader
-        table.find('input:checked').prop('checked', false);
-      } else if ($(this).hasClass('select-all')) {
-        table.find('input:not(:checked)').prop('checked', true);
-        $("#screen-reader-messages").text("All transactions selected."); 
+    clearAllElems.addEventListener("click", function() {
+      const table = selectTable;
+      if (this.classList.contains('clear-all')) {
+        let checkboxes = document.querySelectorAll('input:checked');
+        checkboxes.forEach(function(box) {
+          box.checked = false;
+        })
+        document.querySelector("#screen-reader-messages").textContent = "All transactions deselected.";  //this adds a message to the message div which is read out by the screen reader
+      } else if (this.classList.contains('select-all')) {
+        let checkboxes = table.querySelectorAll('input:not(:checked)');
+        checkboxes.forEach(function(box) {
+          box.checked = true;
+        })
+        document.querySelector("#screen-reader-messages").textContent = "All transactions selected.";
       }
-      resetSelectClearLink($(this).parents('table'))
+      resetSelectClearLink(table);
       return false;
     });
 
-    $('.select-clear-all').keypress(function(ev) {
+    // TODO: test this
+    clearAllElems.addEventListener('keypress', function(ev) {
       const returnKeyCode = 13;
       const spaceBarCode = 32;
-      if (ev.which==returnKeyCode || ev.which==spaceBarCode) { //on space or return
-        $(this).click(); //mimic a click
+      if (ev.which === returnKeyCode || ev.which === spaceBarCode) { //on space or return
+        this.click(); //mimic a click
         return false;
       }
     });
 
-    $('table').find("input").click(function() {
-      resetSelectClearLink($(this).parents('table'));
-      setCategory($(this).parents('tr'))
+    const table = selectTable
```

Comment:
if selectTable has already been decared outside this scope, couldn't you remove this line and just rename 'selectTable' to 'table'